_maps/RandomZLevels/snowdin.dmm

```diff
@@ -638,7 +638,7 @@
 "mr" = (/obj/structure/table/reinforced,/obj/machinery/light{dir = 1},/turf/simulated/floor/plasteel{baseturf = /turf/simulated/floor/plating/asteroid/snow; wet = 0},/area/awaymission/snowdin/sekret)
 "ms" = (/obj/structure/sign/securearea{desc = "A warning sign which reads 'WEAPONS HOT'."; name = "WEAPONS HOT"; pixel_y = 32},/turf/simulated/floor/plasteel{baseturf = /turf/simulated/floor/plating/asteroid/snow; wet = 0},/area/awaymission/snowdin/sekret)
 "mt" = (/obj/item/ammo_casing/c10mm,/obj/item/ammo_casing/c10mm,/turf/simulated/floor/plasteel{baseturf = /turf/simulated/floor/plating/asteroid/snow; wet = 0},/area/awaymission/snowdin/sekret)
-"mu" = (/mob/living/simple_animal/bot/medbot{name = "Accidents Happen"},/turf/simulated/floor/plasteel{baseturf = /turf/simulated/floor/plating/asteroid/snow; wet = 0},/area/awaymission/snowdin/sekret)
+"mu" = (/mob/living/simple_animal/bot/medbot{name = "Accidents Happen", faction=list("syndicate")},/turf/simulated/floor/plasteel{baseturf = /turf/simulated/floor/plating/asteroid/snow; wet = 0},/area/awaymission/snowdin/sekret)
```

Comment:
Yeah we don't control this.
Oddly enough string vars DO get spaces, but lists don't.